API/DTOs/PlayerRanksDTO.cs

```diff
@@ -1,20 +1,67 @@
 namespace API.DTOs;
 
-public class PlayerRanksDTO
+/// <summary>
+/// Represents a player with included osu! ranking info
+/// </summary>
+public class PlayerRanksDTO : PlayerDTO
 {
-    public int Id { get; set; }
-    public long OsuId { get; set; }
+    /// <summary>
+    /// Current standard rank for the player
+    /// </summary>
     public int? RankStandard { get; set; }
+
+    /// <summary>
+    /// Current taiko rank for the player
+    /// </summary>
     public int? RankTaiko { get; set; }
+
+    /// <summary>
+    /// Current catch rank for the player
+    /// </summary>
     public int? RankCatch { get; set; }
+
+    /// <summary>
+    /// Current mania rank for the player
+    /// </summary>
     public int? RankMania { get; set; }
 
-    public int? EarliestOsuGlobalRank { get; set; }
-    public DateTime? EarliestOsuGlobalRankDate { get; set; }
+    /// <summary>
+    /// Earliest highest standard rank available for the player
```

Comment:
```suggestion
    /// Earliest known standard rank available for the player after they started playing tournaments
```

Use similar language throughout